### PR TITLE
feat(Tiktok): Bump compatibility to 32.5.3

### PR DESCRIPTION
**Pull request by d4rkk3y**
Integrations changes: https://github.com/ReVanced/revanced-integrations/pull/536

### CODE REVIEW

**Comment by kazimmt** [comment]
Closes #3244

**Comment by kazimmt** [comment]
Any specific reason why `Hide Ads` patch was removed?

File: src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt [hunk 1]
```diff
@@ -13,8 +13,8 @@ import com.android.tools.smali.dexlib2.Opcode
     name = "Playback speed",
     description = "Enables the playback speed option for all videos.",
     compatiblePackages = [
-        CompatiblePackage("com.ss.android.ugc.trill"),
```
Comment (kazimmt) [inline src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt:16]: It's not even working on 30.8.4
#2814

File: src/main/kotlin/app/revanced/patches/tiktok/feedfilter/FeedFilterPatch.kt [hunk 2]
```diff
@@ -18,8 +18,8 @@ import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
     description = "Filters tiktok videos: removing ads, removing livestreams.",
     dependencies = [IntegrationsPatch::class, SettingsPatch::class],
     compatiblePackages = [
-        CompatiblePackage("com.ss.android.ugc.trill"),
-        CompatiblePackage("com.zhiliaoapp.musically")
+        CompatiblePackage("com.ss.android.ugc.trill", ["32.5.3"]),
+        CompatiblePackage("com.zhiliaoapp.musically", ["32.5.3"])
```
Comment (kazimmt) [inline src/main/kotlin/app/revanced/patches/tiktok/feedfilter/FeedFilterPatch.kt:22]: It's not even working on 30.8.4
#2814

File: src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt [hunk 3]
```diff
@@ -13,8 +13,8 @@ import com.android.tools.smali.dexlib2.Opcode
...
-        CompatiblePackage("com.zhiliaoapp.musically")
+        CompatiblePackage("com.ss.android.ugc.trill", ["30.8.4"]),
+        CompatiblePackage("com.zhiliaoapp.musically", ["30.8.4"])
```
Comment (kazimmt) [inline src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt:17]: It's not even working on 30.8.4
https://github.com/ReVanced/revanced-patches/issues/2814

**Comment by d4rkk3y** [comment]
> Any specific reason why `Hide Ads` patch was removed? 

I think the Feed filter patch is enough to hide ads. If I use Hide Ads patch only without Feed filter tiktok still show ads.

File: src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt (hunk 3 above)
Comment (d4rkk3y) [reply src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt:17]: Do you know the lastest version which that patch still work?

Comment (kazimmt) [reply src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt:17]: Maybe 30.7.xx
But it's too old.
TikTok always shows update popup on this version.

Comment (d4rkk3y) [reply src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt]: nvm, I bumped that patch.

**Comment by CoolCooler0** [comment]
> > Any specific reason why `Hide Ads` patch was removed?
> 
> I think the Feed filter patch is enough to hide ads. If I use Hide Ads patch only without Feed filter tiktok still show ads.

Yes, Hide ads and feed filters patch does the same thing. I'm using v31.2.5 and the hide ads patch always fail when I patch it but I dont get ads.

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**